.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
             PGHOST: 127.0.0.1
         - image: circleci/postgres:10-alpine-ram
           environment:
-            POSTGRES_USER: postgres
+            POSTGRES_USER: circleci
```

Comment:
Sorry totally missed these questions last week. I think I changed this because I had a build that was failing specifically because it complained about the `POSTGRES_USER` not existing, and then I think I might have googled it and found a suggestion to use `circleci` as the user like this yeah? But I can't quite remember now...